README.md

```diff
@@ -197,8 +197,9 @@ within `initializationOptions?: LSPAny;` we support the following settings:
     "medium": true,
     "low": true,
   },
-  "scanningMode": "auto" // Specifies the mode for scans: "auto" for background scans or "manual" for scans on command
-  "authenticationMethod": "token" // Specifies the authentication method to use: "token" for Snyk API token or "oauth" for Snyk OAuth flow. Default is token.
+  "scanningMode": "auto", // Specifies the mode for scans: "auto" for background scans or "manual" for scans on command
+  "authenticationMethod": "token", // Specifies the authentication method to use: "token" for Snyk API token or "oauth" for Snyk OAuth flow. Default is token.
+  "snykCodeApi": "https://deeproxy.snyk.io" // Specifies the Snyk Code API endpoint to use. Default is https://deeproxy.snyk.io
```

Comment:
Why not to name `snykCodeApiUrl` for clarity?

Reply:
I think it's fine :)